sunpy/map/tests/test_mapbase.py

```diff
@@ -1505,8 +1505,6 @@ def test_contour_inputs(simple_map):
     with pytest.raises(ValueError, match='Contour levels must be increasing'):
         simple_map.draw_contours([10, -10] * u.dimensionless_unscaled)
 
-    with pytest.raises(ValueError, match=r'The provided level \(1000.0\) is not smaller than the maximum data value \(8\)'):
-        simple_map.draw_contours(1000 * u.dimensionless_unscaled, fill=True)
```

Comment:
Why does this not error anymore?

Reply:
As the above code is part of `@pytest.mark.skipif(Version(matplotlib.__version__) < Version("3.6.0")` it doesn't run and because now min mpl is `v3.5.0` so no error.
I wonder why it is failing when I used v`3.6.0`

Reply:
Well the skip if needs to go and we need to increase our minimum version of MPL to 3.6.0 as it supports this feature now.

Reply:
Okay.